pkg/app/app_deploy.go

```diff
@@ -122,6 +123,20 @@ func (a *App) inspect() exec.CmdRunResult {
 	return result
 }
 
+// trySaveMetadata if unable to save the kapp metadata into an App meta continue and do not fail the deploy.
+func (a *App) trySaveMetadata(kapp *ctldep.Kapp) {
+	if a.metadata != nil {
+		return
+	}
```

Comment:
nit: i dont think we need this, since we probably would want to always use the "latest".

Reply:
ok removed